packages/site-admin/src/components/page/stories/index.stories.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { Button, __experimentalHStack as HStack } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { settings } from '@wordpress/icons';
+/**
+ * Internal dependencies
+ */
+import { Page } from '../';
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Should move up to `External dependencies`

Reply:
ee group the types under the /* Types js block.
We can move them if we want, but I’d prefer to do that in a follow-up, Lourens.